examples/fluids/tests-output/blasius_stgtest.yaml

```diff
@@ -13,11 +13,6 @@ platemesh:
   growth: 1.3
 
 stab: 'supg'
-Ctau_t: 1
-Ctau_v: 36
-Ctau_C: 0.25
-Ctau_M: 0.25
-Ctau_E: 0.125
 
 q_extra: 0
```

Comment:
It can be a new PR, but shall we make `q_extra = 0` default? It was introduced long ago to handle aliasing as occurs in spectral element methods like Nek. My understanding is that people rarely use over-integration with SUPG because it sufficiently damps the high frequencies. We could preserve the existing unit test output by dropping `-q_extra 2` into their flags.

Reply:
I can fix it in this PR really quick, if you don't mind.

Reply:
Just changed it. I am not sure why not all tests needed the flag though (at least when testing on my local machine).

Reply:
It probably doesn't change the answer enough to matter in most cases. Thanks!